apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -0,0 +1,584 @@
+                    <div class="row">
+                      ${editableCourses &&
+                      editableCourses.length > 0 &&
+                      resLocals.authz_data.has_course_permission_view &&
+                      resLocals.question.course_id === resLocals.course.id
+                        ? html`
+                            <div class="col-auto">
+                              <button
+                                type="button"
+                                class="btn btn-sm btn-primary"
+                                id="copyQuestionButton"
+                                data-toggle="popover"
+                                data-container="body"
+                                data-html="true"
+                                data-placement="auto"
+                                title="Copy this question"
+                                data-content="${escapeHtml(
+                                  copyForm({
+                                    csrfToken: resLocals.__csrf_token,
+                                    exampleCourse: resLocals.course.example_course,
+                                    editableCourses,
+                                    courseId: resLocals.course.id,
+                                  }),
+                                )}"
+                                data-trigger="manual"
+                                onclick="$(this).popover('show')"
+                              >
+                                <i class="fa fa-clone"></i>
+                                <span>Make a copy of this question</span>
+                              </button>
+                            </div>
+                          `
+                        : ''}
+                      ${resLocals.authz_data.has_course_permission_edit &&
+                      !resLocals.course.example_course
+                        ? html`
+                            <div class="col-auto">
+                              <button
+                                class="btn btn-sm btn-primary"
+                                href="#"
+                                data-toggle="modal"
+                                data-target="#deleteQuestionModal"
+                              >
+                                <i class="fa fa-times" aria-hidden="true"></i> Delete this question
+                              </button>
+                            </div>
+                            ${DeleteQuestionModal({
+                              qid: resLocals.question.qid,
+                              a_with_q_for_all_ci,
+                              csrfToken: resLocals.__csrf_token,
+                            })}
+                          `
+                        : ''}
+                    </div>
+                  </div>
+                `
+              : ''}
+          </div>
+        </main>
+      </body>
+    </html>
+  `.toString();
+}
+
+function copyForm({
```

Comment:
As with other pages, these component-ish functions should have `PascalCase` names.